feg/gateway/configs/magmad.yml

```diff
@@ -66,7 +66,6 @@ mconfig_modules:
   - feg.protos.mconfig.mconfigs_pb2
 
 metricsd:
-  log_level: INFO
```

Comment:
I remember that a  couple of years ago we removed log_level from here so it could be controlled by orc8r. 
If you put it here, that value will overwrite whatever is comming from orc8r.

Reply:
The log level inside the service config is not picked up only the one at line 13 in this file.